include/effectmanager.h

```diff
@@ -96,6 +96,7 @@ class EffectManager : public IJSONSerializable
             _clearTempEffectWhenExpired = true;
 
             // This is a hacky way to ensure that we start the correct effect after the temporary one
+            // REVIEW: Explain, I don't get it!

```

Comment:
You know what, I'll just suggest a change to add it here. :)

```suggestion
            //   The switching to the next effect is taken care of by NextEffect(), which starts with
            //   increasing _iCurrentEffect. We therefore need to decrease it here, to make sure that
            //   the first effect after the temporary one is the one we want (either the then current
            //   one when the chip was powered off, or the one at index 0).
```